src/class.rs

```diff
@@ -64,6 +66,8 @@ pub struct ClassPropertyDef {
   pub ts_type: Option<TsTypeDef>,
   pub readonly: bool,
   pub accessibility: Option<deno_ast::swc::ast::Accessibility>,
+  #[serde(skip_serializing_if = "Vec::is_empty")]
+  pub decorators: Vec<DecoratorDef>,
```

Comment:
Does it make sense to skip serializing if vector is empty? We don't do that for any other elements

Reply:
I want to migrate to it in the future, and since this is additive it isn't a breaking change.  Having loads of properties on objects with empty arrays has a non-negligible impact on performance, when you start doing that by thousands ands thousands of nodes that can be in an JSON structure.  Any empty array adds no information and just causes "bloat".  We don't print out `there are no methods` in we format it for display, so it doesn't increase readability or usability.

Reply:
Okay, that's fair.